src/services/services.ts

```diff
@@ -173,7 +173,7 @@ namespace ts {
             const token = scanner.scan();
             const textPos = scanner.getTextPos();
             if (textPos <= end) {
-                if (token === SyntaxKind.Identifier) {
+                if (token === SyntaxKind.Identifier && !isJSDocLink(parent)) {

```

Comment:
Can you explain a bit about why this is needed? That might be worth putting in the code as well

Reply:
the type JSDocComment isn't a node, so *its* children are unexpected here. I am trying to turn comment text into real nodes, but it's not going well so far. The new node needs to be treated as trivia in a number of places that I'm slowly uncovering.

Reply:
Later: JSDoc comments are now real parts of the AST instead of treated as trivia, which has never been quite accurate. This hack is no longer needed.